database/seeds/UsersSeeder.php

```diff
@@ -54,5 +54,9 @@ public function run()
             }
             $user->traders()->sync($trade_ids);
         }
+
+        // So we reliably have one user with a single trader.
+        $trader = App\Trader::find(1);
+        $trader->traders()->sync([1]);
```

Comment:
It syncs the trader to the user with the whole array so any existing others would be wiped out.